vite.config.mts

```diff
@@ -30,17 +30,38 @@ export default defineConfig({
         dts({
             include: ['src'],
         }),
-        externalizeDeps({
-            include: [/^react-is(?:\/.*)?$/, /^@mui(?:\/.*)?$/],
-        }),
     ],
     build: {
         lib: {
             entry: path.resolve(__dirname, 'src/index.js'),
             formats: ['es'],
         },
         rollupOptions: {
+            external: (id: string) =>
+                !id.startsWith('.') && !path.isAbsolute(id),
```

Comment:
It's not clear that the 2nd condition is for the entry file
```suggestion
                !id.startsWith('.') && !id.startsWith(path.resolve(__dirname, 'src')),
```

Reply:
I prefer let `isAbsolute` in case we use absolute path imports one day

Reply:
well ... okay.
but I don't see how an opensource project who can be launch by all can have an absolute path hard-coded in an import 🤔

Reply:
We have it in gridstudy https://github.com/gridsuite/gridstudy-app/blob/d7506d08785b03f08b90dc4f5a039772662289fb/src/redux/reducer.js#L18